doc/examples/animations_pulsating_sphere.py

```diff
@@ -0,0 +1,127 @@
+"""Animations of pulsating sphere."""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation
+
+
+def update_frame_displacement(i, grid, displacement, phasor, scat, frames):
```

Comment:
This is only used once, so what about defining it within `animation_particle_displacement()`?

This way you could capture the variables and avoid specifying `fargs`.